### ISSUE TITLE
question items will be not showed after creating one for a concept

### ISSUE THREAD

**Issue by liuy97** [DASISH/qddt-client#84]
Request
'''
https://qddt.nsd.no/api/concept/page/by-topicgroup/7f000101-54aa-131e-8154-aa293bd10002?page=0&size=20&sort=asc
'''
All of questionitems of the concepts are empty. That means that the front end works, but the backend does not return data.

**Comment by StigNorland** [comment]
There is no questionitem stored in the db.
reason, the save statement has the wrong format, 
[see](https://github.com/DASISH/qddt-client/wiki/QuestionItem)

**Comment by liuy97** [comment]
If you know which code is wrong, please specify it. It works at my local machine. When creating question item, an exception will be  "Content type 'application/json;charset=UTF-8' not supported".

**Comment by liuy97** [comment]
Answer your questions yesterday as follows:
1. Whether did I commit all of my codes?
   Yes, I committed all of my modified codes when I closed the issues. In order to confirm this today, I did "git reset --hard" and change API_BASE to my local IP. It still works on my machine, whether the local backend runs.
2. Whether did I use a wrong format?
   No. But I will merge two steps into one step according to your requirement at next commit.
